Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM sourcegraph/src-cli:3.29.1@sha256:bfdec9e91fdd9d9bac4eab89c9496a9e8e027ffcac0048d56893d3747f8b7da9 AS src-cli
+FROM sourcegraph/src-cli:3.33.5@1b2584614c1013bd4c448e552a317a1e9a0bb6444d4ab0c1e5005facd17da395 AS src-cli
```

Comment:
This line was missing the `sha256:` piece, fixed in https://github.com/sourcegraph/lsif-go/commit/407f5a1d20a6ca2382a6e88cd348c953f56b8215?w=1